apps/condo/domains/contact/utils/serverSchema/index.js

```diff
@@ -5,14 +5,29 @@
  */
 
 const { generateServerUtils } = require('@condo/domains/common/utils/codegeneration/generate.server.utils')
-
+const { GqlWithKnexLoadList } = require('@condo/domains/common/utils/serverSchema')
 const { Contact: ContactGQL } = require('@condo/domains/contact/gql')
 /* AUTOGENERATE MARKER <IMPORT> */
 
 const Contact = generateServerUtils(ContactGQL)
 /* AUTOGENERATE MARKER <CONST> */
 
+const loadContactsForExcelExport = async ({ where = {}, sortBy = ['createdAt_DESC'] }) => {
+    const contactsLoader = new GqlWithKnexLoadList({
+        listKey: 'Contact',
+        fields: 'id name phone email unitName createdAt updatedAt',
+        singleRelations: [
```

Comment:
looks like a static array, could be moved to a constant outside of function

Reply:
Yes, but it's more clear to see code suggestions while writing this argument. Look at `GqlWithKnexLoadList`\`s constructor.

Reply:
I don’t think that it’s increase readability. It’s better to leave it as us